js/utils.js

```diff
@@ -6,7 +6,7 @@ require.scopes.utils = ( () => {
     function extractHostFromURL (url) {
         if (!url) return;
 
-        let urlObj = new URLParser(url);
+        let urlObj = URLParser.parse(url);
```

Comment:
Can you wire this up as `tldjs.parse` so its clear this maps back to npm `tldjs`?